crates/ruff_linter/resources/test/fixtures/pylint/self_or_cls_assignment.py

```diff
@@ -3,7 +3,6 @@ class Fruit:
     def list_fruits(cls) -> None:
         cls = "apple"  # PLW0642
         cls: Fruit = "apple"  # PLW0642
-        cls += "orange"  # PLW0642
```

Comment:
Maybe we should leave the fixture exactly as it is (but change the `# PLW0642` comments to `# OK, augmented assignments are ignored`), so that the snapshot "asserts" that augmented assignments _don't_ trigger the rule

Reply:
Silly me!